addons/edi_notifier/views/notifiers.xml

```diff
@@ -36,6 +36,7 @@
                 <field name="lookback_hours" attrs="{'invisible': ['!', ('model_name', 'ilike', 'In Range')]}"/>
                 <field name="include_notes" attrs="{'invisible': ['!', '|', ('model_name', 'ilike', 'Failed'), ('model_name', 'ilike', 'Success')]}"/>
                 <field name="include_issues" attrs="{'invisible': ['!', ('model_name', 'ilike', 'Failed')]}"/>
+                <field name="include_attachments"/>
```

Comment:
Do we want the same attrs as the other `include_XXX`?

Reply:
I've amended to display for success and failure models (the same as notes) - does that seem reasonable?